pkg/preflight/aws.go

```diff
@@ -64,90 +71,90 @@ func (a *aws) CheckCloudDrivePermission(cluster *corev1.StorageCluster) error {
 		return err
 	}
 	// check the permission here by doing dummy drive operations
-	// TODO: use dry run options
 	logrus.Info("preflight starting eks cloud permission check")
 
-	// Create volume
-	volName := volumePrefix + cluster.Name + "-" + uuid.New()
-	labels := map[string]string{
-		labelVolumeName: volName,
-		labelClusterID:  cluster.Name,
-	}
-	v, err := a.ops.Create(a.getEC2VolumeTemplate(), labels)
+	// List volume on cluster ID first, Describe permission checked here first
+	result, err := a.ops.Enumerate(nil, map[string]string{labelClusterID: cluster.Name}, "")
```

Comment:
Can we use the storagecluster uuid as well for enumerate?
This is a corner case, but there could be two different px clusters (different k8s clusters) getting deployed at the same time with the same name.

Reply:
sure, good point, I'm adding another label, so it's easier to filter on name as well, which is more readable.

Reply:
sorry the cluster name is already in the volume name, I'll use cluster UID only.